index.html

```diff
@@ -0,0 +1,1543 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/main.css">
+    <title>Twitter</title>
+</head>
+
+<body>
+    <div class="container ">
+        <header class="pd-sd-md">
+            <ul class="nav">
```

Comment:
a ul with a class "nav", perhaps should have been within a nav tag?

Reply:
I needed to use the `ul` tag for the `li` tags to works properly. But I coudl maybe replace the `header` tag with a nav one since it isn't really a header. Thanks !